autoload/neomake/makers/ft/go.vim

```diff
@@ -47,3 +47,19 @@ function! neomake#makers#ft#go#govet()
             \ '%-G%.%#'
         \ }
 endfunction
+
+let g:neomake_go_gometalinter_args =
+    \ get(g:, 'neomake_go_gometalinter_args', ['--disable-all',
+        \ '--enable=varcheck', '--enable=structcheck', '--enable=ineffassign',
+        \ '--enable=unconvert', '--enable=gosimple', '--enable=staticcheck'])
```

Comment:
`neomake_go_gometalinter_args` should work already out of the box.
You just need to specify the defaults below with the `'args'` key.